akka-actor-typed-tests/src/test/java/jdocs/akka/typed/InteractionPatternsTest.java

```diff
@@ -896,6 +938,23 @@ public void timers() throws Exception {
     probe.expectMessage(Duration.ofSeconds(2), new Buncher.Batch(Arrays.asList(msgOne, msgTwo)));
   }
 
+  @Test
+  public void testIgnoreReply() throws Exception {
+    // #ignore-reply-doit
+    final ActorSystem<PingPong.Ping> system =
+        ActorSystem.create(PingPong.create(), "ping-pong-system");
+
+    // note that system is also the ActorRef to the guardian actor
+    final ActorRef<PingPong.Ping> ref = system;
+
+    // sends a message using 'tell' and ignores the reply, fire-and-forget style
+    ref.tell(new PingPong.Ping(ActorRef.ignore()));
+    // #ignore-reply-doit
+
+    system.terminate();
+    system.getWhenTerminated().toCompletableFuture().get(5, TimeUnit.SECONDS);
```

Comment:
Can use `ActorTestKit.shutdown(...); instead. That will also await the termination. I don't know why other places here don't use it. Good if you change all.